trl/trainer/dpo_trainer.py

```diff
@@ -69,6 +66,136 @@
     import deepspeed
 
 
+def _tokenize(
+    example: Dict[str, Any],
+    tokenizer: "PreTrainedTokenizerBase",
```

Comment:
To be consistent with other implementations, we can allow `tokenizer` to be a processor here.